src/main/daml/Daml/Finance/Interface/Holding/Base.daml

```diff
@@ -41,19 +22,12 @@ data View = View
       -- ^ Key of the account holding the assets.
     amount : Decimal
       -- ^ Size of the holding.
-    lock : Optional Lock
-      -- ^ When a contract is locked, contains the locking details.
   deriving (Eq, Show)
 
 -- | Base interface for a holding.
-interface Base requires Disclosure.I where
+interface Base requires Lockable.I, Disclosure.I where
```

Comment:
As per the comment above ^^^ :

```suggestion
interface Base requires Disclosure.I, Lockable.I where
```

Reply:
If we agree that we should be ordered, I'll let you deal with these in the other related `interface`s.

Reply:
see previous comment about ordering according to requires